cni/network/network.go

```diff
@@ -452,6 +452,7 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) error {
 		EnableInfraVnet:    enableInfraVnet,
 		PODName:            k8sPodName,
 		PODNameSpace:       k8sNamespace,
+		SkipHotAttachEp:    false,
```

Comment:
add a comment, why false, similar to cnm.

Reply:
Done